api_app/core/cloud.py

```diff
@@ -0,0 +1,27 @@
+from msrestazure.azure_cloud import AZURE_PUBLIC_CLOUD, AZURE_US_GOV_CLOUD
+from core import config
+
+
+def get_cloud():
+    arm_env = config.ARM_ENVIRONMENT.lower()
+    supported_clouds = {"public": AZURE_PUBLIC_CLOUD, "usgovernment": AZURE_US_GOV_CLOUD}
+    if arm_env in supported_clouds:
+        return supported_clouds[arm_env]
+    raise ValueError(f"Invalid arm environment. Got: {arm_env}. Supported envs are: {', '.join(supported_clouds.keys())}.")
+
+
+def get_authority():
+    return get_cloud().endpoints.active_directory.replace("https://", "")
+
+
+def get_aad_authority():
+    return get_cloud().endpoints.active_directory
```

Comment:
If I had to think what is the difference between authority and aad_authority....
Can we think about better names?